chef/cookbooks/neutron/templates/default/dnsmasq-neutron.conf.erb

```diff
@@ -0,0 +1,4 @@
+# this file is managed by chef to give extra options to dnsmasq
+<% if @low_mtu -%>
+dhcp-option-force=26,1458
```

Comment:
I found the 1458 by testing with ssh and netcat transfers. size calculation might be IP+Eth+VLAN+GRE = 20 + 14 + 4 + 4
we can give it another 40 bytes to be on the safe (but slower) side